src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/MS/Internal/Automation/AutomationIdentifierConstants.cs

```diff
@@ -31,7 +31,15 @@ internal static class AutomationIdentifierConstants
 
         static AutomationIdentifierConstants()
         {
-            if (OSVersionHelper.IsOsWindows10RS2OrGreater)
+            if (OSVersionHelper.IsOsWindows10RS4OrGreater)
```

Comment:
UIA team says IsDialog was new in RS5.  So you need an RS5 clause.

Reply:
Done